test/cypress/integration/group-large.spec.js

```diff
@@ -16,14 +16,12 @@ describe('Payments', () => {
     })
 
     cy.giLogout()
-  })
 
-  it('20 users join the group', () => {
     for (let i = 2; i <= 21; i++) {
       cy.giAcceptGroupInvite(invitationLinks.anyone, {
```

Comment:
One thing about how Cypress works when dealing with async functions:

Now this test never will fail because `invitationLinks.url` is null. You need to split back this test into 2 parts (create creation + joining group). Here's  [the explanation about it](https://stackoverflow.com/questions/59401325/cypress-store-a-value-to-be-accessed-later)